### PR TITLE
Unified the size of headers and sb-line-headers

### LINKED ISSUE #971: h1 in transclusion is styled more prominently than h1 in containing page
See screenshot:

Screenshot 2024-07-23 at 9 45 30 am

Both `Week` and `Goals` are `h1`s

### PR DESCRIPTION
**Pull request by roceb**
Fixes #971

### CODE REVIEW

File: web/styles/editor.scss [hunk 1]
```diff
@@ -49,6 +49,23 @@
   .sb-line-h6 {
     font-size: 1em;
   }
+  h1 {
```
Comment (zefhemel) [inline web/styles/editor.scss:52]: Could you merge this with .sb-line-h1?

```
.sb-line-h1, h1 {
```

**Review by zefhemel — CHANGES_REQUESTED** [review]
Minor request to merge definitions (for all)

File: web/styles/editor.scss (hunk 1 above)
Comment (roceb) [reply web/styles/editor.scss:52]: My first implementation was `.sb-line-h1, h1 `  and it didn't work locally

Comment (roceb) [reply web/styles/editor.scss]: Well I redid it and it worked.

**Comment by zefhemel** [comment]
Perfect, thanks!

**Pull request merged**